### ISSUE TITLE
EPGs Disappeared?

### ISSUE THREAD

**Issue by boeder8** [acidjesuz/EPG#3]
Hi Jesus.

Love your work! Been relying on your EPG guide for months.

I noticed that the epgs for many US and Mexico channels have disappeared lately (last 3 days or so). I can't find the channels on your guide even when I attempt to re-link on my TiviMate.

Were these removed by any chance?

Hopefully it's by error, I see the newest update xml is pretty small.

Please add all the channels you had, it's truly hard to replace the quality of your guide.

God bless,

boeder

**Comment by acidjesuz** [comment]
Hey boeder,

I didn't change anything on my side. No channels were removed.
It had to be an error in pulling the guide from the source. Let me do a fresh pull and upload to git and see if that fixes the problem.

aCiD

**Comment by acidjesuz** [comment]
Hey boeder8,

Please check now.

aCiD

**Comment by acidjesuz** [comment]
Hey boeder8,

Yeah, I didn't even notice it was only a 1.75MB of a pull.. it's all automated so unless someone let's me know, sometimes I can miss it...it happens once in a while ;)   Good Morning to You sir, hope you'll have a great week ahead of you.

I'm still in the weekend time zone.  

Take care there.

aCiD

**Comment by boeder8** [comment]
> Hey boeder8,
> 
> Yeah, I didn't even notice it was only a 1.75MB of a pull.. it's all automated so unless someone let's me know, sometimes I can miss it...it happens once in a while ;) Good Morning to You sir, hope you'll have a great week ahead of you.
> 
> I'm still in the weekend time zone.
> 
> Take care there.
> 
> aCiD

No worries buddy, totally understandable.

I'll refresh the epg on tivimate later in the day and report back ASAP - but I think the missing channels should be there given the size of the XML.

Enjoy the few hours of weekend left and God bless!

**Comment by s-digweed** [comment]
> Hey boeder8,
> 
> Please check now.
> 
> aCiD

Yup, I can confirm that the channels EPGs are back and running. Many thanks

**Comment by boeder8** [comment]
> Hey boeder8,
> 
> Please check now.
> 
> aCiD

Hello aCiD,

Yup I can confirm that the channel EPGs are back and running.

Many thanks!

**Comment by skydrome** [comment]
Think it might have messed up again? Only a megabyte again

**Comment by boeder8** [comment]
> Think it might have messed up again? Only a megabyte again

**Comment by boeder8** [comment]
> > Think it might have messed up again? Only a megabyte again

Yeah looks like it, my player hasn't updated the EPGs yet but when it will the channels will disappear again lol

**Comment by s-digweed** [comment]
> Think it might have messed up again? Only a megabyte again

LMAO a bad file again. Looks like it only works if you do it twice 🤣

**Comment by boeder8** [comment]
> Think it might have messed up again? Only a megabyte again

It was fine after you fixed it yesterday but LMAO a bad file again just a while ago. Looks like it only works if you do it twice 🤣

**Comment by acidjesuz** [comment]
Hey guys,

It looks like only happens to midday pull at noon. All other pulls are ok. EPG get's updated every six hours at midnight, six am, noon, and at six pm CST. I'll change the pull to 1 pm today and we'll see if that helps any. 

If not I'll have to investigate more over the weekend to see why is this happening.

aCiD

**Comment by boeder8** [comment]
> Hey guys,
> 
> It looks like only happens to midday pull at noon. All other pulls are ok. EPG get's updated every six hours at midnight, six am, noon, and at six pm CST. I'll change the pull to 1 pm today and we'll see if that helps any.
> 
> If not I'll have to investigate more over the weekend to see why is this happening.
> 
> aCiD

No problem, thanks buddy!
